src/app/views/company/company.controller.coffee

```diff
@@ -17,7 +17,8 @@ angular.module 'mnoEnterpriseAngular'
         if vm.isBillingShown()
           vm.activeTab = 'billing'
         else
-          vm.activeTab = 'members'
+      #On refreshing the page it should stay on the same tab
```

Comment:
Use one space between the leading # character of the comment and the text of the comment.